frontend/src/components/Hero.tsx

```diff
@@ -0,0 +1,27 @@
+import { useTranslation } from "next-i18next";
+import { GridContainer } from "@trussworks/react-uswds";
+
+const Hero = () => {
+  const { t } = useTranslation("common", {
+    keyPrefix: "Hero",
+  });
+
+  return (
+    <div
+      data-testid="hero"
+      className="bg-primary desktop:padding-y-15 tablet:padding-y-2 padding-y-1"
+    >
+      <GridContainer>
+        <h1 className="text-white desktop:font-sans-3xl tablet:font-sans-2xl font-sans-xl">
+          {t("title")}
+        </h1>
+        <div className="desktop:font-sans-xl tablet:font-sans-lg font-sans-md">
+          <span className="text-yellow text-bold">{t("beta")}</span>&nbsp;
+          <span className="text-white">{t("content")}</span>
```

Comment:
```suggestion
          <span className="usa-intro text-white">{t("content")}</span>
```

Based on this comment from the figma, but really up to @andycochran for guidance.
![image](https://github.com/HHS/grants-equity/assets/25465464/6bfac6a3-0ced-441b-a49b-8cab63f97db7)

Reply:
Meh. Class semantics. I don't think it really matters which classes we use to achieve this. A `p` might be more semantic than the `div`, but not sure that matters all that much either. If we were to use the `.usa-intro` class, it should probably go on the span's container so the size gets applied the "beta" too. `</nit>`

Reply:
Changing the parent `div` to a `p` tag and then adding `usa-intro` looks great! Went ahead and committed that, thank you.